### DISCUSSION TITLE
PagImageView部分手机动画显示不出来，但listener的方法均正常调用

### DISCUSSION THREAD

**Discussion by jhforestever** [Tencent/libpag#1634] [Q&A]
Android SDK 4.2.100，问题如题。

**Comment by kevingpqi123** [comment]
仅仅一个描述无法定位问题，建议提供复现的 demo 工程、复现机型和复现步骤

**Comment by jhforestever** [comment]
oppo F11 pro 目前有这种情况，另外pag文件是本地加载循环播放，pagImageView是作为自定义布局View的一部分加载的，initView时即加载pag文件。目前初步判断应该是加载pag文件时view还没有attachedWindow，实际上可能还和具体的CPU内核或手机调度策略有关。

**Reply by kevingpqi123** [reply]
为了提升对接效率，请提供复现 demo 工程及复现路径

**Comment by kevingpqi123** [comment]
未能提供更多的信息，暂时关闭
